x-pack/plugins/index_management/public/application/sections/home/home.tsx

```diff
@@ -92,24 +99,32 @@ export const IndexManagementHome: React.FunctionComponent<RouteComponentProps<Ma
         <EuiSpacer size="m" />
 
         <EuiTabs>
-          {tabs.map(tab => (
+          {tabs.map(({ id, name, ...rest }) => (
             <EuiTab
-              onClick={() => onSectionChange(tab.id)}
-              isSelected={tab.id === section}
-              key={tab.id}
-              data-test-subj={`${tab.id}Tab`}
+              onClick={() => onSectionChange(id)}
+              isSelected={id === section}
+              key={id}
+              {...rest}
             >
-              {tab.name}
+              {name}
             </EuiTab>
           ))}
         </EuiTabs>
 
         <EuiSpacer size="m" />
 
         <Switch>
-          <Route exact path={`${BASE_PATH}indices`} component={IndexList} />
-          <Route exact path={`${BASE_PATH}indices/filter/:filter?`} component={IndexList} />
-          <Route exact path={`${BASE_PATH}templates/:templateName*`} component={TemplateList} />
+          <Route exact path={`${BASE_PATH}${Section.Indices}`} component={IndexList} />
+          <Route
+            exact
+            path={`${BASE_PATH}${Section.Indices}/filter/:filter?`}
+            component={IndexList}
+          />
+          <Route
+            exact
+            path={`${BASE_PATH}${Section.IndexTemplates}/:templateName*`}
```

Comment:
Just noticed this will change the path from `base_path/templates` to `base_path/index_templates`. Should we add a release note for this (for example, a user could have previously bookmarked this page)?